app/views/users/new.html.erb

```diff
@@ -3,6 +3,7 @@
 
   <div class="row center">
     <%= form_for @user, class: "col s12" do |f| %>
+    <%= render 'users/error_messages', object: f.object %>
```

Comment:
@cmacaulay This is slightly different from the pattern I've seen before (in the past have seen the errors been captured in the controller, and then iterate over them in the view), but I think this pretty much does what you need it to do.

One thing: recommend not displaying this if there are no errors (maybe I'm missing it, but it seems like it's always displaying whether or not there are any errors).